### ISSUE TITLE
Query about Combobox

### ISSUE THREAD

**Issue by nzneil22** [lylejohnson/fxruby#9]
Hi,

Is there a way to open the menu pane of a Combobox using code?

example, FXComboBox.showMenu(TRUE)

TIA

**Comment by damian-m-g** [comment]
Try to invoke the event that would open that. It has been a long time since I don't do that, but I think you could do something like this:

`@widget.handle(@widget, FXSEL(SEL_COMMAND, 0), nil)`

That would trigger the SEL_COMMAND event on certain @widget, in your case, the #FXComboBox. Try to see what event open it.

Hope this helps somehow. Cheers.

**Comment by nzneil22** [comment]
Hi, thanks for your reply.

That is not exactly what i am looking for, let me rephrase.

I would like to toggle visibility of the combobox's selection drop down list

Thanks.

**Comment by nzneil22** [comment]
Thanks for your reply.

I could not find a particular event that opens the menu of the combo box unfortunately

Cheers
